src/main/resources/config.sk

```diff
@@ -338,6 +338,11 @@ databases:
 
 		backup interval: 2 hours
 
+		backup purge: false #set to "true" to enable the automatic purging feature
```

Comment:
maybe explain what purging does exactly on prior lines instead of on the same line